apps/myjobs/config/configuration_singleton.rb

```diff
@@ -97,8 +97,8 @@ def production_database_path
     Pathname.new(ENV["DATABASE_PATH"] || dataroot.join('production.sqlite3')).expand_path
   end
   
-  def whitelist_paths
-    ENV['WHITELIST_PATH'].to_s.strip.split(":")
+  def allowlist_paths
+    (ENV['WHITELIST_PATH'] + ENV['ALLOWLIST_PATH']).to_s.strip.split(":")
```

Comment:
I think we should alias the old configs, not keep both or outright replace. That way we keep backwards compatibility (which is super important) but we can revamp our docs to only use the new configuration and we don't run into undesirable behaviour where both are defined. 

Which is to say, these should be `||` instead of `+` like `( ENV['NEW'] || ENV['OLD'] ).to_s`